src/main/java/core/basesyntax/FigureSupplier.java

```diff
@@ -0,0 +1,32 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class FigureSupplier {
+    private static final double MAX_NUM = 10;
+    private static final int NUM_OF_FIGURES = 5;
+    private final Random random = new Random();
+    private final ColorSupplier color = new ColorSupplier();
+
+    public Figure getRandomFigure() {
+        switch (random.nextInt(NUM_OF_FIGURES)) {
+            case 0:
+                return new Square(color.getRandomColor(), random.nextDouble() * MAX_NUM);
```

Comment:
```suggestion
        switch (random.nextInt(NUM_OF_FIGURES)) {
            case 0:
                return getRandomSquare();
```
lets add private methods to get each random figure - to make code easier to read

Reply:
thank you